### DISCUSSION TITLE
Unable to open directory when using code-server on build environments

### DISCUSSION THREAD

**Discussion by techiemac** [blinksh/blink#1773] [Support] [answered]
### Checklist

- [x] Using latest version on the [App Store](http://itunes.apple.com/app/id1156707581)
- [x] Read the [docs](http://blink.sh/docs/)
- [x] Searched for [existing GitHub issues](https://github.com/blinksh/blink/issues)

### Configuration

iPad Pro version 16.4.1(a)
Blink version v16.1.7.748 (App Store)
VS  Server version 20220909.27

### Describe the bug

I launched a Blink Build environment using `build mosh`. The build environment was built out using the video here https://docs.blink.sh/advanced/code. I pulled down a git repo on the blink build environment.
I launch `code-server` and then go to another blink shell and launch `code https://ENV_DNS_ADDR` (note, full name hidden).
When I go to Add Folder, I can navigate to the repo folder but when I click Ok, the folder is not added to vscode. I’ve tried installing Blink-FS but this plugin only installs locally and not on the code-server. [This issue](https://github.com/blinksh/blink/issues/1440) implies Blink-FS needs to be installed on the code-server target instance to make this but this doesn’t appear to be possible. Unless my `code-server`-fu is missing something :).

I can open the folder using https://ENV_DNS_ADDR:/root/somecode and I get asked to trust the folder. At first I thought this was related to establishing folder trust so I removed the folder and re-added it but I still could not add it via the vscode UI.

If there is something I’m missing here, some guidance or a bug fix would be good. Or do I manually need to upload the vsix extension to the Blink Build environment and manually install it?

**Comment by carloscabanero** [comment, marked as answer]
Hi! It is a stupid thing but the issue is on VS Code side. Instead of clicking in the OK button, press enter. They have this quirks here and there (with syncing for example, it is the opposite). 

No Blink-FS is needed for this.

**Comment by techiemac** [comment]
That was the issue! Thanks! 
Any chance this could be added to the documentation?
Also, do you know if there is a VS Code issue open for this?

**Reply by mabeledo** [reply]
I run a self-hosted instance. The issue is reproducible only in Blink. When I try this in a browser, it works just fine.
